packages/form-js-editor/src/rendering/PropertiesPanel.js

```diff
@@ -411,13 +426,15 @@ function ValidationGroup(field, editField) {
         field,
         id: 'minLength',
         label: 'Minimum Length',
+        min: 0,
         path: [ 'validate', 'minLength' ]
       }),
       NumberEntry({
         editField,
         field,
         id: 'maxLength',
         label: 'Maximum Length',
+        min: 0,

```

Comment:
Does a max length of 0 sense? Shouldn't it be at least 1?

Reply:
Well, does a maximum length of 1 make sense? 😄 We shouldn't make any assumptions. Negative lengths are a bug but a length of 0 is not.

Reply:
Makes sense. 👍